docs/main/blog/2019-05-29-long-blog-post.md

```diff
@@ -0,0 +1,44 @@
+---
+slug: long-blog-post
+title: Long Blog Post
+authors: endi
+tags: [hello, docusaurus]
+---
+
+This is the summary of a very long blog post,
```

Comment:
U wanna keep it as a template?

Reply:
I replied here days ago, but it somehow got lost. I thought it wouldn't hurt to keep in, to showcase the blogging feature which I think is pretty sick